UPGRADE-7.2.md

```diff
@@ -8,6 +8,11 @@ Read more about this in the [Symfony documentation](https://symfony.com/doc/7.2/
 
 If you're upgrading from a version below 7.1, follow the [7.1 upgrade guide](UPGRADE-7.1.md) first.
 
+FrameworkBundle
+---------------
+
+ * The `secrets:decrypt-to-local` command terminates with a non-zero exit code when a secret could not be read
```

Comment:
Do we want to mark this as BC break, for better visibility?

```suggestion
 * [BC BREAK] The `secrets:decrypt-to-local` command terminates with a non-zero exit code when a secret could not be read
```

Reply:
done